src/components/dashboard/SendLinkSummary.js

```diff
@@ -155,6 +156,16 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
           userStorage.markWithErrorEvent(txHash)
         },
       })
+      const { txPromise } = generateLinkResponse
+
+      txPromise.catch(e => {
+        log.error(e.message, e)
+        showErrorDialog('Could not complete transaction. Please try again.', '', {
```

Comment:
@AlexeyKosinski please consolidate error message

Reply:
@sirpy 
`Send Link generation failed. Please try again`
is it a suitable message?

Reply:
review errors, make sure they make sense, try to use same error messages and format where possible